<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>890</width>
    <height>563</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>0</y>
      <width>871</width>
      <height>461</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;img src=&quot;:/new/photo_2.jpg&quot;/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>890</width>
     <height>30</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu_S">
    <property name="title">
     <string>系统(&amp;S)</string>
    </property>
    <addaction name="action_L"/>
    <addaction name="action_R"/>
    <addaction name="action_M"/>
    <addaction name="action_X"/>
   </widget>
   <widget class="QMenu" name="menu_S_2">
    <property name="title">
     <string>通信(&amp;S)</string>
    </property>
    <addaction name="action_S"/>
    <addaction name="action_C"/>
    <addaction name="action_F"/>
    <addaction name="action_V"/>
    <addaction name="action_E"/>
   </widget>
   <widget class="QMenu" name="menu_H">
    <property name="title">
     <string>帮助(&amp;H)</string>
    </property>
    <addaction name="action_H"/>
    <addaction name="action_F_2"/>
    <addaction name="separator"/>
    <addaction name="action_A"/>
   </widget>
   <addaction name="menu_S"/>
   <addaction name="menu_S_2"/>
   <addaction name="menu_H"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="action_L"/>
   <addaction name="action_R"/>
   <addaction name="action_S"/>
   <addaction name="action_H"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="action_L">
   <property name="icon">
    <iconset>
     <normaloff>:/new/pic/images/登录.png</normaloff>:/new/pic/images/登录.png</iconset>
   </property>
   <property name="text">
    <string>登陆(&amp;L)</string>
   </property>
  </action>
  <action name="action_R">
   <property name="icon">
    <iconset>
     <normaloff>:/new/pic/images/注册.PNG</normaloff>:/new/pic/images/注册.PNG</iconset>
   </property>
   <property name="text">
    <string>注册(R)</string>
   </property>
  </action>
  <action name="action_M">
   <property name="icon">
    <iconset>
     <normaloff>:/new/pic/images/修改.PNG</normaloff>:/new/pic/images/修改.PNG</iconset>
   </property>
   <property name="text">
    <string>修改密码(&amp;M)</string>
   </property>
  </action>
  <action name="action_X">
   <property name="text">
    <string>退出(&amp;X)</string>
   </property>
  </action>
  <action name="action_C">
   <property name="icon">
    <iconset>
     <normaloff>:/new/pic/images/收发字符.PNG</normaloff>:/new/pic/images/收发字符.PNG</iconset>
   </property>
   <property name="text">
    <string>收发字符(&amp;C)</string>
   </property>
  </action>
  <action name="action_F">
   <property name="icon">
    <iconset>
     <normaloff>:/new/pic/images/收发文件.PNG</normaloff>:/new/pic/images/收发文件.PNG</iconset>
   </property>
   <property name="text">
    <string>收发文件(&amp;F)</string>
   </property>
  </action>
  <action name="action_V">
   <property name="icon">
    <iconset>
     <normaloff>:/new/pic/images/查看文件.PNG</normaloff>:/new/pic/images/查看文件.PNG</iconset>
   </property>
   <property name="text">
    <string>查看文件(&amp;V)</string>
   </property>
  </action>
  <action name="action_H">
   <property name="icon">
    <iconset>
     <normaloff>:/new/pic/images/帮助.PNG</normaloff>:/new/pic/images/帮助.PNG</iconset>
   </property>
   <property name="text">
    <string>查看帮助(&amp;H)</string>
   </property>
  </action>
  <action name="action_F_2">
   <property name="icon">
    <iconset>
     <normaloff>:/new/pic/images/意见反馈.PNG</normaloff>:/new/pic/images/意见反馈.PNG</iconset>
   </property>
   <property name="text">
    <string>意见反馈(&amp;F)</string>
   </property>
  </action>
  <action name="action_A">
   <property name="icon">
    <iconset>
     <normaloff>:/new/pic/images/关于系统.PNG</normaloff>:/new/pic/images/关于系统.PNG</iconset>
   </property>
   <property name="text">
    <string>关于系统(&amp;A)</string>
   </property>
  </action>
  <action name="action_S">
   <property name="icon">
    <iconset>
     <normaloff>:/new/pic/images/通信设置.PNG</normaloff>:/new/pic/images/通信设置.PNG</iconset>
   </property>
   <property name="text">
    <string>通信设置(&amp;S)</string>
   </property>
  </action>
  <action name="action_E">
   <property name="text">
    <string>串口通讯设置(E)</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
